airflow/providers/fab/auth_manager/security_manager/override.py

```diff
@@ -470,7 +470,9 @@ def reset_password(self, userid, password):
         user = self.get_user_by_id(userid)
         user.password = generate_password_hash(password)
         self.reset_user_sessions(user)
-        self.update_user(user)
+        if self.update_user(user) is False:
```

Comment:
Can we update update_user to return True  in case of success?

Reply:
Besides the True/False comment above, why not directly returning the result, if the result is `False` returning `False` is more than needed.

Reply:
I'd done the if false check since None returned by update_user would also be interpreted as false, changing it to return a true or false should resolve this as well.